### DISCUSSION TITLE
Are links to localized content missing from the REST API?

### DISCUSSION THREAD

**Discussion by LorensoD** [statamic/cms#5735] [Troubleshooting] [answered]
I have a multisite but can't seem to find a way to get the [Locales Tag](https://statamic.dev/tags/locales) information in the REST API.

I have a page that is translated to 3 languages. However, the API doesn't seem to return any data about the translated versions of this page. Which I can get with the locales tag. The only thing that I can find in the API response which is related to locales is the following:

```json
"locale": "en",
"localized_urls": [],
```

But as you can see, the localized_urls property is empty. Am I missing something here?

**Comment by afonic** [comment, marked as answer]
I don't think the Content API has that information. You can probably add it by overriding the class: 
https://statamic.dev/rest-api#customizing-resources

**Reply by LorensoD** [reply]
Thanks for the tip! I currently fixed it by creating the following custom resource:

```php
<?php

namespace App\Http\Resources;

use Statamic\Http\Resources\API\EntryResource;

class CustomEntryResource extends EntryResource
{
    public function toArray($request)
    {
        return array_merge([
            'descendants' => $this->resource->descendants()->toAugmentedArray(),
        ], $this->resource
            ->toAugmentedCollection()
            ->withShallowNesting()
            ->toArray());
    }
}
```

Weird that this information is not accessible through the Content API by default.

**Comment by jasonvarga** [comment]
How are you getting `localized_urls` in there at all? That's not something provided by Statamic.

**Reply by LorensoD** [reply]
It seems to be coming from the Aardvark SEO plugin.

**Reply by jasonvarga** [reply]
Then I'd suggest making an issue on their repo.
